src/settings_gui.cpp

```diff
@@ -688,7 +688,7 @@ struct GameOptionsWindow : Window {
 			case WID_GO_BASE_GRF_DROPDOWN:
 				if (_game_mode == GM_MENU) {
 					CloseWindowByClass(WC_GRF_PARAMETERS);
-					auto* set = BaseGraphics::GetSet(index);
+					auto set = BaseGraphics::GetSet(index);
```

Comment:
These two in this file lost their pointer. Does `auto` figure that out automatically?

Reply:
Yes, with the caveat that including it ensures you are dealing with a pointer. Probably an unnecessary distinction in most cases.